app/views/catalog/_bibliography_default.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='col-md-12 bibliography-contents'
+     data-path="<%= search_exhibit_catalog_path(exhibit_id: @exhibit) -%>"
+     data-parentid="<%= @document.id -%>"
+     data-sort="author_sort asc, pub_year_isi asc, title_sort asc">
```

Comment:
I wonder if we can derive this value from the `blacklight_config`somehow, or at the very least throw this into a configuration so this isn't hard-coded into the partial.

Just commenting on the potential fragility, we don't necessarily need to do anything about it now.